pages/join.html

```diff
@@ -31,6 +31,13 @@ <h2 id="instructors">Instructors</h2>
   to update your information.
 </p>
 
+<h2 id="discussion">Community Events</h2>
+<p>
+  You could meet online with others contributors in many opportunities.
+  For that, check the calendar below.
```

Comment:
Replace with: "There are many opportunities to join community lab meetings, sub-committees and de/briefing sessions. Subscribe to the below calendar to watch all that is going on throughout our community."